banyand/index/index.go

```diff
@@ -31,7 +31,7 @@ type Condition struct {
 }
 
 type Repo interface {
-	Search(indexName string, startTime, endTime uint64, conditions []Condition) ([]common.ChunkID, error)
+	Search(index common.Metadata, startTime, endTime uint64, conditions []Condition) ([]common.ChunkID, error)
```

Comment:
Would you complete this `Condition` struct in this PR or maybe the following ones?

Reply:
I would like to left it to you to complete it .